mgmtfn/k8splugin/kubeClient_test.go

```diff
@@ -367,8 +378,16 @@ func setupTestServer(c *C) {
 	tlsCfg := &tls.Config{
 		Certificates: []tls.Certificate{cert},
 		RootCAs:      caPool,
+		ClientAuth:   tls.VerifyClientCertIfGiven,
+		ClientCAs:    caPool,
 	}
 	tlsCfg.BuildNameToCertificate()
+	// Read the token
+	var contivK8Config ContivConfig
+
+	err = getConfig(testCfgFile, &contivK8Config)
+	token = contivK8Config.K8sToken
+
```

Comment:
@jojimt, This can be controlled with the configuration file - which now provides both the https://github.com/neelimamukiri/netplugin/blob/master/mgmtfn/k8splugin/certs/contiv.json.
Changing the config file used for testing, you can test both these cases as well as the no certs/token cases.
Do you want me to add a config specific test invocation in https://github.com/contiv/netplugin/blob/master/scripts/unittests?